File: app/src/androidTest/java/ru/org/adons/cblock/suite/DataModelTestSuite.java
package ru.org.adons.cblock.suite;

import org.junit.runner.RunWith;
import org.junit.runners.Suite;

import ru.org.adons.cblock.data.BlockListModelTest;
import ru.org.adons.cblock.data.CallLogModelTest;

/**
 * Suite to run all tests together
 */

@RunWith(Suite.class)
@Suite.SuiteClasses({
        CallLogModelTest.class,
        BlockListModelTest.class
})
public class DataModelTestSuite {
}

File: app/src/androidTest/java/mobile/addons/cblock/suite/DataModelTestSuite.kt
package mobile.addons.cblock.suite

import org.junit.runner.RunWith
import org.junit.runners.Suite

import mobile.addons.cblock.data.BlockListModelTest
import mobile.addons.cblock.data.CallLogModelTest

/**
 * Suite to run all tests together
 */
@RunWith(Suite::class)
@Suite.SuiteClasses(CallLogModelTest::class, BlockListModelTest::class)
class DataModelTestSuite
